tests/VnV/vnv-onboarding-phase/Makefile

```diff
@@ -0,0 +1,22 @@
+# DOCKER TASKS
+
+build: ## Build the image
+		docker image build -t docker-robot .
+
+build-nc: ## Build the image without caching
+		docker image build --no-cache -t docker-robot .
+
+run: ## Run container
+		docker container run -d --rm -i  --net=host --name docker-robot-container --mount type=bind,src=//c/Users/asinatra/Desktop/ECLIPSE_WORKSPACE/tng-tests/tests/VnV/vnv-onboarding-phase/tests,dst=/docker-robot/tests docker-robot 
```

Comment:
@sinaure I think this route would not work in Jenkins since is a local route in your PC

Reply:
Try to use a relative path instead absolute path

Reply:
can't... mount path must be absolute...
anyway the mount path was needed just for dev phase when I wan to run robot and modify the files and conf.
To run it locally during dev will be 'make run-dev'
To run it on jenkins will be 'make run'

Reply:
Do I need to close and reopen the PR after pushing changes?